config/application.rb

```diff
@@ -1,28 +1,25 @@
 require_relative "boot"
 
 require "rails"
-# Pick the frameworks you want:
-require "active_model/railtie"
-require "active_job/railtie"
-require "active_record/railtie"
-require "active_storage/engine"
-require "action_controller/railtie"
-require "action_mailer/railtie"
-require "action_mailbox/engine"
-require "action_text/engine"
-require "action_view/railtie"
-require "action_cable/engine"
-# explicitly not loading 'sprockets/railtie'
-require "rails/test_unit/railtie"
+require "rails/all"
```

Comment:
**not a blocker**, but the indvidual component give us more fine grained control i would have thought  - for example are we using action_cable and action_mailbox?

Reply:
Generally speaking most Rails apps have `rails/all`, see [cccd](https://github.com/ministryofjustice/Claim-for-Crown-Court-Defence/blob/master/config/application.rb#L3)

We only swicthed to fine-grained to remove sprockets when we switched to (or tried to) WebPacker
Sub gems still required sprockets eventually 😢 

This just reverts to the standard rails import